R/await.R

```diff
@@ -124,13 +125,14 @@ await_all <- function(f, .x, ...,
   start <- Sys.time()
   fname <- as.character(substitute(f))
 
-  repeat {
-    responses[!called] <- lapply(.x[!called], safe_call_once,
-                                 f = f, ..., .status_key = .status_key,
-                                 .success_states = .success_states,
-                                 .error_states = .error_states,
-                                 fname = fname)
+  zipped_parameters <- if (is.null(.y)) .x else as.list(paste(.x, .y))
```

Comment:
Do we want `as.list(paste(.x, .y))` here or `mapply(list, .x, .y, SIMPLIFY=FALSE)`?  I think the former will concatenate the parameters into a string, while the latter gives us a list of tuples that can be passed to `safe_call_once`.

Reply:
I changed it to lapply(strsplit(paste(.x, .y), ' '), as.integer), which might be more what we want...?

> mapply(list, .x, .y, SIMPLIFY=FALSE)
[[1]]
[[1]][[1]]
[1] 1

[[1]][[2]]
[1] 3


[[2]]
[[2]][[1]]
[1] 2

[[2]][[2]]
[1] 4

> lapply(strsplit(paste(.x, .y), ' '), as.integer)
[[1]]
[1] 1 3

[[2]]
[1] 2 4

Reply:
Hoho, I think you're right we want the output with the `strsplit`.  We should be able to do that without needing to do the back and forth between strings and integers with `mapply(c, x, y, SIMPLIFY=FALSE)`.  (i.e. using `c` instead of `list` in `mapply`).